src/qibolab/instruments/qm.py

```diff
@@ -393,21 +393,32 @@ def classify_shots(self):
 
 
 class QMPulse:
-    def __init__(self, pulse):
-        self.pulse = pulse
-        self.element = f"{pulse.type.name.lower()}{pulse.qubit}"
-        self.operation = pulse.serial
-        self.relative_phase = pulse.relative_phase / (2 * np.pi)
-        self.duration = pulse.duration
-        # cycles to wait before playing the pulse
-        # this is calculated and assigned by :meth:`qibolab.instruments.qm.Sequence.add`
-        self.wait_cycles = None
-
-        self.acquisition = None
-
-        # Stores the baking object (for pulses that need 1ns resolution)
+    """Wrapper around :class:`qibolab.pulses.Pulse` for easier translation to QUA program."""
+
+    def __init__(self, pulse: Pulse):
+        self.pulse: Pulse = pulse
+        """:class:`qibolab.pulses.Pulse` implemting the current pulse."""
```

Comment:
Is this recognized by Sphinx?

I know this is working for class attributes, but it is the first time I see inside a method.
Perfectly possible, I'm just surprised.

Reply:
I don't think it is recognized, but I have not checked. I mainly did it for the code to look more uniform. In principle I could change this to a dataclass to be the same as the other classes around it, however I do not think dataclass is a good choice in this  case because we are calculating most of the attributes in `__init__`

Reply:
Or otherwise I could just move these docstrings at the definition of the class, which is for sure recognized by Sphinx.

Reply:
You can also keep as they are, we'll use them the moment this will transition to a dataclass :)